Library/Homebrew/brew.sh

```diff
@@ -23,7 +23,9 @@ if [[ -n "$HOMEBREW_MACOS" ]]
 then
   HOMEBREW_DEFAULT_CACHE="${HOME}/Library/Caches/Homebrew"
   HOMEBREW_DEFAULT_LOGS="${HOME}/Library/Logs/Homebrew"
+  HOMEBREW_DEFAULT_PREFIX="/usr/local"
   HOMEBREW_DEFAULT_TEMP="/private/tmp"
+  HOMEBREW_MACOS_ARM_DEFAULT_PREFIX="/opt/homebrew"
```

Comment:
If these stay around given other comments after this:

- they need to be used in `global.rb` so that their definitions aren't duplicated
- a definition should be present for Linux